### PR TITLE
Regenerated fixtures, with imports from multiple sources

### PR DESCRIPTION
**Pull request by exonian**

### CODE REVIEW

**Comment by exonian** [comment]
@meshy All fixtures regenerated, and a quicker way to load them all (which also doesn't go out of date in the readme quite so quickly!)

File: cbv/management/commands/load_all_django_versions.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class Command(BaseCommand):
+    """Load the Django project fixtures and all version fixtures"""
+
+    def handle(self, **options):
+        fixtures_dir = os.path.join(settings.DIRNAME, 'cbv', 'fixtures')
+        self.stdout.write('Loading project.json')
+        call_command('loaddata', 'cbv/fixtures/project.json')
+        version_fixtures = [re.match(r'((?:\d+\.){2,3}json)', filename) for filename in os.listdir(fixtures_dir)]
```
Comment (ghickman) [inline cbv/management/commands/load_all_django_versions.py]: Use [glob](https://docs.python.org/2/library/glob.html#glob.glob) here to save the matching and try/catch block.

File: cbv/management/commands/load_all_django_versions.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        for match in version_fixtures:
+            try:
+                fixture = match.group()
+            except AttributeError:
+                continue
+            self.stdout.write('Loading {}'.format(fixture))
+            call_command('loaddata', 'cbv/fixtures/{}'.format(fixture))
```
Comment (ghickman) [inline cbv/management/commands/load_all_django_versions.py]: You could use `os.path.join` here too, it's not really necessary though.

Comment (exonian) [inline cbv/management/commands/load_all_django_versions.py]: Yeah I thought about it but decided that `loaddata` was probably handling turning it into a valid path anyway. Maybe it doesn't, but I'd never seen mention of having to call it differently on Windows or whatever.

Comment (ghickman) [inline cbv/management/commands/load_all_django_versions.py]: Good enough for me 👍

**Comment by exonian** [comment]
@ghickman glob ftw! :grin:

**Comment by ghickman** [comment]
Nice work!

**Comment by exonian** [comment]
Cheers @ghickman

**Pull request merged**